conans/test/command/config_test.py

```diff
@@ -11,20 +11,11 @@ class ConfigTest(unittest.TestCase):
 
     def setUp(self):
         self.client = TestClient()
-        self.settings_defaults = """arch = x86_64
-build_type = Release
-compiler = Visual Studio
-compiler.runtime = MD
-compiler.version = 14
-os = Windows
-"""
-        client_conf = default_client_conf + self.settings_defaults
-        save(self.client.paths.conan_conf_path, client_conf)
 
     def basic_test(self):
         # show the full file
         self.client.run("config get")
-        self.assertIn("arch = x86_64", self.client.user_io.out)
+        self.assertIn("default_profile = conan_default", self.client.user_io.out)
```

Comment:
I hope this doesn't break to users that are using ``conan config set arch=...`` in CI to customize their defaults